source/index.ts

```diff
@@ -1,5 +1,6 @@
-import * as sourceMapSupport from "source-map-support";
-sourceMapSupport.install();
+// NOTE -- Commented out to avoid cross origin error when run via webpack
```

Comment:
Do we have any additional details like a GitHub issue with Webpack or source-map-support projects?  If so, I would like to get them linked here.  If not, can we edit the comment to say something like, "More research is necessary to investigate the issue" or even better some brief repro steps?